packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,230 @@
+import { Badge } from 'terra-clinical-item-view/package.json?dev-site-package';
+import { Notice } from "@cerner/terra-docs";
+import ItemViewWithHeader from '../example/ItemViewWithHeader?dev-site-example';
+import ItemViewTwoColumnWithHeader from '../example/ItemViewTwoColumnWithHeader?dev-site-example';
+
+<Badge />
+
+# Accessibility Guide for Terra Clinical Item View
+
+## Why is this important?
+
+Terra Clinical Item View allows you to create a view filled with information in the form of Clinical Item Displays. Improper usage of this component may prevent end users from understanding content and necessary context to interact with the page. 
+
+## Accessibility Considerations
+
+#### General notes
+- **Related information should be presented as close to each other as possible.** It is best practice to keep related information above or below its related counterparts.
+  - For example - presenting related information across from each other in the two column view can make it more difficult for users to understand that the information is related.
+Reading by column from top to bottom is the more common way information is consumed visually, so when presenting this information by row between the two columns it might be confusing for both sighted and non-sighted users.
+- When passing in displays, keep in mind that only 8 can be used within a view at a time. Any additional displays passed in will be ignored.
```

Comment:
Yeah that makes sense especially since it's early on in the doc, changed it to be the actual name of the display component: https://github.com/cerner/terra-clinical/pull/916/commits/5117762dc6b5ef7e8b4056b22c0a99646b862df7#diff-bc5256d2671d1c03594dbeb9ddfffd9514706650fe9a7a330c26c0c02d080e10R20